modules/protect/blocked-login-page.php

```diff
@@ -282,7 +282,7 @@ function protect_die( $content, $back_link = false ) {
 			$content = "<style>html{ background-color: #fff; } #error-page { margin:0 auto; padding: 1em; box-shadow: none; } </style>" . $content;
 		}
 
-		wp_die( $content, $this->page_title, array( 'back_link' => $back_link ) );
+		wp_die( $content, $this->page_title, array( 'back_link' => $back_link, 'response' => 200 ) );
```

Comment:
Nice catch, but aren't we dropping `wp_die()` altogether in #8274 ?

Reply:
We are, but this PR will be useful if #8274 does not make it to 5.6.

Reply:
true, let's merge this @oskosk